### ISSUE TITLE
Safe to leave but cannot leave the game.

### ISSUE THREAD

**Issue by exec-dkail** [ValveSoftware/Dota2-Gameplay#3910]
### Description

Played turbo, someone abandoned so it was safe to leave, as usual I tried to leave and start a new turbo game but there was no option. Everytime I click on the Leave Game button the dialog with "Do you really wish to quit dota 2" pop's up but irrespective of whether I select yes or cancel, dota 2 quits automatically.

### Example Match ID (and possibly Timestamp)

6815039572

**Comment by mikelei1997** [comment]
+1 having this issue now, "Leave Game" button was red and disabled

**Comment by mastercoms** [comment]
6824702254

Terrorblade was shown as connected at the top hero bar, however chat kept counting down his DC until he got an abandon and the game became safe to leave and no stats recorded. Now we cannot leave, the Yes, leave game button is red and disabled after disconnecting (in the confirmation window).

**Comment by bezhdeyire** [comment]
same
![image](https://user-images.githubusercontent.com/90929097/198153002-15a9a05f-6caf-406b-aafc-56d53f4911c5.png)

**Comment by aharonk** [comment]
![image](https://user-images.githubusercontent.com/13606487/198340396-3407c4b1-3d69-4a97-be8c-64218427e439.png)
+1 as well

**Comment by bostonapp** [comment]
same here

**Comment by exec-dkail** [comment]
> +1 having this issue now, ~"Leave Game" button was red and disabled~ I just learned that we need to wait for a few seconds until it can be clicked. But my game crashed when I clicked cancel and tried to click the LEAVE GAME button on the main screen the leave the game.

I waited like 20mins :)

**Comment by ShootingStar91** [comment]
I've had this multiple times, I play Turbo only. Sometimes has happened even after game finished successfully, but now happened when a player abandoned and game became safe to leave.

**Comment by jerrymouse1995** [comment]
same here

**Comment by ZiruiYan** [comment]
same problem

**Comment by exec-dkail** [comment]
This is fixed.
closing issue.

**Comment by OliverParson** [comment]
Thanks for the information!

**Comment by Koonus** [comment]
Are there any professional players here? Which casino would you recommend for beginners?

**Comment by Xander355** [comment]
> Are there any professional players here? Which casino would you recommend for beginners?

It's simple. You only need to choose a safe casino and be very careful. You must have reasonable steps and your own strategy. I started gambling more than three years ago, and to be honest, I'm always lucky. I have tested a lot of casinos and games, but most of all I remember aviator game. When I go to https://1xbetonline.in/aviator/ I'm constantly getting nostalgic. I remember the time when my friends and I gathered and played. Now my friends are far away, but one day we will get together and play together again.
